style.css

```diff
@@ -0,0 +1,153 @@
+
+body {
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+    background-color: black;
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
+/* *************************************************************** */
+                /*  all items in header below */
+/* *************************************************************** */
+header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 15%;
+    font-size: 2rem;
+    color: red;
+    font-family: 'Russo One', sans-serif;
+}
+
+h2{
+    height: 6vh;
+    font-size: 10vmin;
+}
+
+
+#playerSwap .button{
+    box-sizing: border-box;
+    align-items: flex-end;
+    clear: left;
+    height: 5vh;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 3%;
+}
+
+
+input {
+    background-color: red;
+    color: black;
+    cursor: pointer;
+    font-size: 24px;
+}
+
+input.switch {
+    background-color: black;
+    color: red;
+}
+
+/* *************************************************************** */
+            /*  all items in game board below */
+/* *************************************************************** */
+ .container {
+    /*height: 72%;*/
+    display: flex;
+    justify-content: center;
+}
+
+#gameBoard {
+    height: 75vmin;
+    width: 69vmin;
+}
+
+.row {
+    height: 12%;
+    width: 100%;
+    background-color: black;
+}
+
+.col {
+    height: 100%;
+    width: 14%;
+    float: left;
+    box-sizing: border-box;
+    border-radius: 50%;
+    background-color: yellow;
+    border: 2px solid white;
+    animation: drop 1s;
+}
+
+@keyframes drop {
+    from {
+        transform: translate3d(0, -80vmin, 0);
```

Comment:
Why use translate3d if you are only tanslating in the Y axis?  Were you going for the other benefit of using the 3d variant?